package.json

```diff
@@ -37,10 +38,9 @@
     "gh-pages": "^2.0.0",
     "immutable": "4.0.0-rc.15",
     "json-rules-engine": "^4.0.0",
-    "json-schema-ref-parser": "^7.1.1",
     "katex": "0.11.1",
     "lodash-es": ">=4.17.14",
-    "markdown-it": "^11.0.0",
+    "markdown-it": "^12.3.2",
```

Comment:
Fixes critical vulnerability

Reply:
Are the other markdown-it plugins still compatible (did you try testing the math renderer in markdown, for example)?

Reply:
Images, YouTube embeds, and TeX all still work. The main breaking changes in markdown-it 12 seem to relate to code highlighting and a markdown table edge case, so I think it's a pretty low risk. I don't think those features are used much where we use our React markdown renderer.